cmd/cluster-operator-controller-manager/app/controllermanager.go

```diff
@@ -385,20 +390,23 @@ func GetAvailableResources(clientBuilder controller.ClientBuilder) (map[schema.G
 // controllers such as the clientBuilder.
 func CreateControllerContext(s *options.CMServer, clientBuilder controller.ClientBuilder, stop <-chan struct{}) (ControllerContext, error) {
 	versionedClient := clientBuilder.ClientOrDie("shared-informers")
+	kubeClient := clientBuilder.KubeClientOrDie("shared-informers")
```

Comment:
Just curious if this name here needs to be unique for any reason?

Reply:
it's just the string in the user agent that will show up in logs afaik